versions.yml

```diff
@@ -1,12 +1,7 @@
 ---
 
-aerogear_mobile_services: "2.0.0-alpha2"
-mobile_developer_console_apb: "1.0.4"
-unifiedpush_apb: "1.0.3"
-keycloak_apb: "1.2.2"
-metrics_apb: "1.1.2"
-sync_app_apb: "1.1.1"
-rhcc_release_tag: "1.0"
+mobile_services_namespace: "mobile-developer-services"
```

Comment:
Can we remove this here as it's listed in the namespace role defaults?

Reply:
The idea is that this could be overriden by users. But yeah, it shouldn't be in this file. Will move it to a different one.